packages/cli/src/createApp.ts

```diff
@@ -246,16 +246,23 @@ export async function createApp(
       resolve(cwd(), `./${appName}/src/main.${options.lang}`),
       buildMain()
     )
+
+    green(`Created ${appName}!
+
+      To run your new app:
+      📁 cd ${appName}
+      ✅ npm install
+      🚀 npm run dev
+    `)
   } else {
     options.lang = 'ts'
     info('Fetching nuxi cli...')
     const subprocess = execaCommand(
-      `npx --yes nuxi@latest init --no-install $APP_NAME`,
+      `npx --yes nuxi@latest init --no-install ${appName}`,
```

Comment:
One issue here is that if the appName contains spaces this will not work properly. We could use quotes I suppose, or perhaps do some kind of escaping strategy

Reply:
The changes now make sure its surrounded by " so it works for names with space and folders.